taxonomies/food/categories.txt

```diff
@@ -33557,6 +33557,7 @@ fr:Sticks à la Mozzarella, Batonnets de Mozzarella
 ja:モッツァレラスティック
 nl:Mozzarella sticks
 wikidata:en:Q1055940
+minimum_number_of_ingredients:en:3
```

Comment:
This is not mozzarella but Mozzarella sticks.

Please move to the simple Mozzarella entry !